growth/too/models.py

```diff
@@ -945,6 +947,164 @@ class Observation(db.Model):
          comment='processed successfully?')
 
 
+class Observations(db.Model):
```

Comment:
Should be singular for consistency with other classes.

Reply:
We already have an 'Observation' class in models.py, what is a more appropriate name? @mcoughlin any suggestion?

Reply:
OK, have changed this to ObservationList